src/fragments/lib-v1/storage/js/get-properties.mdx

```diff
@@ -0,0 +1,66 @@
+You can get file properties and metadata without downloading the file using `getProperties` API from `aws-amplify/storage`
+
+```javascript
+import { getProperties } from 'aws-amplify/storage';
+
+try {
+	const result = await getProperties({
+		key: 'key',
+		options: {
+			accessLevel: 'guest', // defaults to `guest` but can be 'private' | 'protected' | 'guest'
+			targetIdentityId: 'xxxxxxx', // id of another user, if `accessLevel: protected`
+		},
+	});
+	console.log('File Properties ', result);
+} catch (error) {
+	console.log('Error ', error);
+}
+```
+
+The format of the response will look similar to the below example
+
+```js
+ {
+    key : "key",
+    contentType: "image/jpeg",
+    contentLength: 6873,
+    eTag: "\"56b32cf4779ff6ca3ba3f2d455fa56a7\"",
+    lastModified: Wed Apr 19 2023 14:20:55 GMT-0700 (Pacific Daylight Time) {},
+    metadata: { owner : 'aws'}
+}
+
+```
+
+<Callout>
+To get the metadata in result for all read apis you have to configure user defined metadata in CORS
+
+Update your bucket's CORS Policy to look like:
+
+```json
+[
+  {
+    "AllowedHeaders": [
+      "*"
+    ],
+    "AllowedMethods": [
+      "GET",
+      "HEAD",
+      "PUT",
+      "POST",
+      "DELETE"
+    ],
+    "AllowedOrigins": [
+      "*"
+    ],
+    "ExposeHeaders": [
+      "x-amz-server-side-encryption",
+      "x-amz-request-id",
+      "x-amz-id-2",
+      "ETag",
+      "x-amz-meta-test-owner" // user defined metaData
+     ],
+  }
+]
+```
+Learn more about how to setup a [CORS Policy](https://docs.amplify.aws/lib-v1/storage/getting-started/q/platform/js/#amazon-s3-bucket-cors-policy-setup)
+</Callout>
```

Comment:
Nit: this exists in both download and getProperties. Can it just be a link to cors-polify-setup section in the getting started page?